corehq/apps/app_manager/static/app_manager/js/app_manager.js

```diff
@@ -9,6 +9,13 @@
     COMMCAREHQ.app_manager.checkCommcareVersion = function (version) {
         return COMMCAREHQ.app_manager.versionGE(COMMCAREHQ.app_manager.commcareVersion(), version);
     };
+    COMMCAREHQ.app_manager.checkCommcareVaporWare = function (version) {
+        if (COMMCAREHQ.app_manager.latestCommcareVersion() === "0") {
```

Comment:
This could now change to `!COMMCAREHQ.app_manager.latestCommcareVersion()` if we use `null` as the way of denoting that the version is absent